package.json

```diff
@@ -1,26 +1,30 @@
 {
   "name": "mozilla-universal-search-addon",
-  "version": "1.0.0",
   "description": "universal search desktop experiments in addon format",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mozilla/universal-search-addon"
-  },
-  "author": "",
-  "license": "MPL2",
+  "version": "1.0.0",
+  "author": "Mozilla (https://mozilla.org/)",
   "bugs": {
     "url": "https://github.com/mozilla/universal-search-addon/issues"
   },
-  "homepage": "https://github.com/mozilla/universal-search-addon",
   "dependencies": {
     "jpm": "^1.0.0"
   },
+  "devDependencies": {
+    "eslint": "0.23.0"
+  },
   "engines": {
     "firefox": ">=38.0a1",
     "fennec": ">=38.0a1"
+  },
+  "homepage": "https://github.com/mozilla/universal-search-addon",
+  "license": "MPL-2.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mozilla/universal-search-addon"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Sorry, this file is a bit noisy, I ran **fixpack** which ends up sorting a lot of the keys from A-Z.
Only changes I made were:

1. Specified `author` value (was empty string).
2. Added **eslint** devDependency. This will get removed when/if we want to switch to Grunt or Gulp and add more tasks.
3. Tweaked `license` value to be SPDX compliant string.
4. Added `lint` task to `scripts` section.